src/ert/_c_wrappers/enkf/enkf_fs.py

```diff
@@ -171,6 +248,34 @@ def copy_from_case(
             nodes,
             active,
         )
+        self._copy_parameter_files(other, nodes, [i for i, b in enumerate(active) if b])
```

Comment:
Probably worth adding some tests to this

Reply:
There exist a test which runs copy_case in test_case_tool.py

Reply:
Should perhaps have a test with multiple parameters? Think in general the test foundation is a bit shaky for multiple parameters/responses